src/sentry/lang/native/plugin.py

```diff
@@ -254,43 +254,23 @@ def run_symbolicator(self, processing_task):
         assert len(self.images) == len(rv['modules']), (self.images, rv)
 
         for image, fetched_debug_file in zip(self.images, rv['modules']):
-            status = fetched_debug_file.pop('status')
+            statuses = [
+                fetched_debug_file.pop(k) for k in (
```

Comment:
_nit_: Can we move this in a constant for legibility?

Reply:
done